src/lava/magma/runtime/runtime.py

```diff
@@ -220,7 +226,7 @@ def _run(self, run_condition):
                                     actors.join()
                                     if actors.exception:
                                         _, traceback = actors.exception
-                                        print(traceback)
+                                        self.log.debug(traceback)
                                         error_cnt += 1
 
                                 raise RuntimeError(
```

Comment:
This message even says that there should be some output above when an error occurs. But if users don't know they have to run logging on then this won't mean much to them.

Reply:
Yes. I missed some of the context. Changing this to log.error()

Reply:
log.error will show on default loglevel of warn.